tez-mapreduce/src/main/java/org/apache/tez/mapreduce/committer/MROutputCommitter.java

```diff
@@ -78,6 +79,9 @@ public void initialize() throws IOException {
     jobConf.getCredentials().mergeAll(UserGroupInformation.getCurrentUser().getCredentials());
     jobConf.setInt(MRJobConfig.APPLICATION_ATTEMPT_ID,
         getContext().getDAGAttemptNumber());
+    jobConf.set(MRJobConfig.JOB_COMMITTER_UUID, Utils.getDAGID(
```

Comment:
this logic is used twice, so it should be refactored to util like:
```
jobConf.set(MRJobConfig.JOB_COMMITTER_UUID, AnyUtilClassYouPrefer.getJobCommitterUuid(getContext()));
```

Reply:
They're actually 2 different context types (OutputCommitterContext vs OutputContext) in those 2 places, but I can create an overloaded method for that, sure.